src/promise-katas-2.js

```diff
@@ -47,7 +56,21 @@ const dog = () => {};
 // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all
 //
 
-const joke = () => {};
+const joke = () => {
+  const question = fetch("jokes", "question");
+  const answer = fetch("jokes");
+
+  const response = Promise.all([question, answer])
+  .then((values) =>  { 
+    const question = values[0];
+    const answer = values[1];
+    const questionKey = "joke";
+    const answerKey = "answer";
+    
+    return { question: question[questionKey], answer: answer[answerKey] }
+  });
+  return response;
```

Comment:
This works, but it can be simplified as you dont really need to create all these variables. By substituting the values for the variables and returning the value of the `Promise.all` function you could refactor this to:
```
  return Promise.all([question, answer]).then((values) => {
    return { question: values[0]["joke"], answer: values[1]["answer"] };
  });
```